### ISSUE TITLE
Send an email when feed is imported and ready to go (or fails)

### ISSUE THREAD

**Issue by rmglennon** [transitland/transitland-datastore#326]
When feed has been imported and is ready to go, send an email to alert the user. Should it also include the changes that were made in the review process?

Work with @meghanhade to get the experience right here.

**Comment by rmglennon** [comment]
@drewda, maybe something like:

Thank you for adding a feed to the Transitland Feed Registry! Your feed has been reviewed and was successfully imported. You can find it in the listings at https://transit.land/feed-registry/.

Contributions from community members like you help us build Transitland. If you know of other feeds, please submit them, too!

If you have any questions, you can contact the Transitland team at [email address link].

**Comment by rmglennon** [comment]
If it fails...perhaps more manual?

Thank you for adding a feed to the Transitland Feed Registry! Your feed has been reviewed, but we were unable to import it to Transitland. 

[Give reason? This usually occurs because the feed does not fully support the standards and expected GTFS files].

If you have any questions, you can contact the Transitland team at [email address link].

**Comment by drewda** [comment]
@rmglennon the Datastore will now send an e-mail notification when a changeset is created and when it is applied.

The text and HTML versions of the e-mail bodies live in: https://github.com/transitland/transitland-datastore/tree/master/app/views/changeset_mailer

And the subject lines for the two e-mails are set in: https://github.com/transitland/transitland-datastore/blob/master/app/mailers/changeset_mailer.rb

Feel free to open pull requests against any of those files to edit the copy.

I didn't include an error state e-mail, because there are a variety of places where an error could pop up.

**Comment by rmglennon** [comment]
@drewda - Those templates look great. Nice work.

**Comment by rmglennon** [comment]
Although it doesn't seem like the application template covers the scenario where the import process is completely done? Maybe I'm not sure about the difference between creation and deletion?

**Comment by drewda** [comment]
@rmglennon yeah, these should be reasonably straightforward to edit in the future.

I had to fix configuration to get e-mails to send on staging #345 but that's working now. And once I started working, I realized that I had accidentally swapped the two templates #347 ;)

You're right, the "application" just means that the changeset has been applied, and the feed and operator records have been created. They'll both appear in the Feed Registry at that point.

It's a little harder for us to cover the import steps that happen next. Those are manually triggered by us. Also, we don't have a trail from the contributing user all the way through the import process (although we will soon #338 )

For now, we might just have an import success and error e-mail "templates" that we copy, paste, and send manually?
